Makefile

```diff
@@ -22,6 +24,20 @@ help:
 	@echo ""
 	@echo "Running 'make' without a target is equivalent to running 'make build run'."
 
+## dev: Run the excretor in development mode
+dev:
+	@echo "Starting tummy with exposed port"
+	@ZIPFILE_PATH=. $(DOCKER_COMPOSE) up tummy-dev -d --wait
+	@echo ""
+	@echo "Starting excretor in development mode. Please run 'make dev-stop' to stop the tummy later."
+	@$(EXCRETOR_DEV_ENVS) cargo run --manifest-path excretor/Cargo.toml
```

Comment:
`--manifest-path` will not work because `styles.css` is loaded from the PWD. Using `pushd` and then running `cargo run` should fix this.